qiskit/pulse/schedule.py

```diff
@@ -327,80 +308,77 @@ def _apply_filter(self, filter_func: Callable, new_sched_name: str) -> 'Schedule
 
     def _construct_filter(self, *filter_funcs: List[Callable],
                           channels: Optional[Iterable[Channel]] = None,
-                          instruction_types=None,
+                          instruction_types: Optional[Iterable[Type['Instruction']]] = None,
                           time_ranges: Optional[Iterable[Tuple[int, int]]] = None,
                           intervals: Optional[Iterable[Interval]] = None) -> Callable:
-        """Returns a boolean-valued function with input type ``(int, ScheduleComponent)`` that
-        returns ``True`` iff the input satisfies all of the criteria specified by the arguments;
-        i.e. iff every function in ``filter_funcs`` returns ``True``, the instruction occurs on a
-        channel type contained in ``channels``, the instruction type is contained in
-        ``instruction_types``, and the period over which the instruction operates is fully
-        contained in one specified in ``time_ranges`` or ``intervals``.
+         """
+        Returns a boolean-valued function with input type (int, ScheduleComponent) that returns True
+        iff the input satisfies all of the criteria specified by the arguments; i.e. iff every
+        function in filter_funcs returns True, the instruction occurs on a channel type contained
+        in channels, the instruction type is contained in instruction_types, and the period over
+        which the instruction operates is fully contained in one specified in time_ranges or
+        intervals.
 
         Args:
             filter_funcs: A list of Callables which take a (int, ScheduleComponent) tuple and
-                          return a bool.
-            channels: For example, ``[DriveChannel(0), AcquireChannel(0)]``.
-            instruction_types (Optional[Iterable[Type[Instruction]]]): For example,
-                ``[PulseInstruction, AcquireInstruction]``.
-            time_ranges: For example, ``[(0, 5), (6, 10)]``.
-            intervals: For example, ``[(0, 5), (6, 10)]``.
+                          return a bool
+            channels: For example, [DriveChannel(0), AcquireChannel(0)] or DriveChannel(0)
+            instruction_types: For example, [PulseInstruction, AcquireInstruction] or DelayInstruction
+            time_ranges: For example, [(0, 5), (6, 10)] or (0, 5)
+            intervals: For example, [Interval(0, 5), Interval(6, 10)] or Interval(0, 5)
         """
-        def only_channels(channels: Set[Channel]) -> Callable:
-            def channel_filter(time_inst) -> bool:
-                """Filter channel.
 
-                Args:
-                    time_inst (Tuple[int, Instruction]): Time
-                """
+        def if_scalar_cast_to_list(to_list):
```

Comment:
Hi @taalexander,
I went through most of the errors in the checks, and all that I saw pointed to an indentation error on this line. The error is: IndentationError: unindent does not match any outer indentation level

I'm sorry for such a simple problem. I tried to re-upload, edit from a previous edition of the file, but none seem to satisfy the standard. Do you know what's going on here?

Reply:
Is it possible that line 314 has ane extra space?

Reply:
Indentation is by 4 spaces, right? I'm pretty sure that the lines are indented correctly. This is quite interesting. Maybe I could do a regex search to see if I might have mixed tabs and spaces? What's the syntax for it?